layers/+intl/keyboard-layout/README.org

```diff
@@ -148,6 +149,23 @@ will remap keybindings for the =dvorak-programmer= variant.
 
 [[file:img/dvorak-keymap.png]]
 
+** workman
+=workman= is an English-optimized keyboard layout designed to,
```

Comment:
@TheBB 

Have I updated the pull request correctly? This is my first time contributing, so I don't know exactly what I'm doing, and don't know if the PR is still pending because I've done something wrong, or because I'm supposed to explicitly notify you via @ , or if you just haven't gotten to it yet. I updated via an amended commit with a forced push, as described [here:](https://github.com/syl20bnr/spacemacs/wiki/Beginner%27s-Guide-to-Contributing-a-Pull-Request-to-Spacemacs)

I certainly don't mean to sound pushy, so please don't take it that way. There's no rush or anything, I just want to be sure I've done things right. :)